src/axolotl/cli/utils.py

```diff
@@ -318,7 +319,8 @@ def load_model_and_tokenizer(
     tokenizer = load_tokenizer(cfg)
 
     LOG.info("loading model...")
-    model, _ = load_model(cfg, tokenizer, inference=inference)
+    model_loader = ModelLoader(cfg, tokenizer, inference=inference)
```

Comment:
For anyone curious about this change, I didn't really like the `load_model` function which was basically a straight pass through to a `ModelLoader(...).load_model()` call.